### ISSUE TITLE
Support CockroachDB as source

### ISSUE THREAD

**Issue by BirknerAlex** [yugabyte/yb-voyager#795]
Jira Link: [DB-13467](https://yugabyte.atlassian.net/browse/DB-13467)
We would like to migrate our existing CRDB to Yugabyte but currently I can't find any way to export the database in a compatible way.

Are there any plans to support the migration of the schema and data using voyager?

**Comment by FranckPachot** [comment]
@BirknerAlex Yes, CRDB doesn't provide reliable export to CSV. Here is a solution, reading data through the Foreign Data Wrapper:
https://dev.to/franckpachot/moving-data-from-cockroachdb-to-postgresql-or-yugabytedb-462h

**Comment by hbhanawat** [comment]
Being tracked as an IDEA. Closing this issue.
